app/web/README.md

```diff
@@ -60,6 +60,8 @@ In the project directory, you can run:
 Runs the app in the development mode.<br />
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
+*hint*: You can find a set of users for logging in at the [dummy data loaded in the docker container](https://github.com/Couchers-org/couchers/blob/develop/app/backend/src/data/dummy_users.json)
+
```

Comment:
Good idea! I think it's better to move this under the `Option 1: ...` heading instead, since these accounts won't be available if you target the preview API instead.

Reply:
Yeah, makes sense. Done - see next commit.